public/app/features/alerting/unified/components/rule-editor/alert-rule-form/simplifiedRouting/AlertManagerRouting.tsx

```diff
@@ -26,12 +27,26 @@ export function AlertManagerManualRouting({ alertManager }: AlertManagerManualRo
   const [selectedContactPointWithMetadata, setSelectedContactPointWithMetadata] = useState<
     ContactPointWithMetadata | undefined
   >();
+  const { watch } = useFormContext<RuleFormValues>();
+
+  const contactPointInForm = watch(`contactPoints.${alertManagerName}.selectedContactPoint`);
+  const { contactPoints } = useContactPointsWithStatus({
```

Comment:
We can instead use `useGetGrafanaContactPoint` to only fetch the one that we need?

Reply:
actually, in the form we fetch all the contact points in the contact point selector. And as it's through RTKQ, we only fetch once...so there is no additional request in the form